Templates/Common-Backend-Scripts/dbbBuild.sh

```diff
@@ -503,26 +505,24 @@ if [ $rc -eq 0 ]; then
 
     ## Except for the reset mode, check for "nothing to build" condition and throw an error to stop pipeline
     if [ "$Type" != "--reset" ]; then
-
-      # Locate the most recent Build Log Directory within outDir. The Build Log Directories are Time Stamped.
-      # Therefore, the last directory entry will be the most recent Build Log. The directory will always be
-      # be created by DBB, but "buildList.txt" may not.  If not created, array will will be blank.
-      array=$(find ${outDir} -name "buildList.txt")
-      for log in ${array[@]}; do
-        LastBuildLog=${log}
-        echo $PGM": [INFO] LastBuildLog = ${LastBuildLog}"
+      
+      # Locate buildList and deletedFilesList in Build Log Directory within outDir, and group them in logListArray
+      buildListFile="${outDir}/buildList.txt"
+      deletedFilesListFile="${outDir}/deletedFilesList.txt"
+      logListArray=(${buildListFile} ${deletedFilesListFile})
```

Comment:
Hi @lauren-li  - How does this behave when the `deletedFilesListFile` does not exist? the buildList.txt is a standard output if the build had run, the deletedFileList.txt is not present always. 

I think we should check if the file exists.

Reply:
Hi @dennis-behm, thank you for looking over the PR! Line 517 (`if [ -f ${list} ]; then`) checks if the filepath exists and is a regular file, before proceeding any further with processing it. If the filepath fails the check (e.g. `deletedFilesListFile` does not exist), then no further processing occurs with that filepath.

Reply:
I missed that. That's cool. Nice code!